src/pseudopeople/configuration/validator.py

```diff
@@ -272,6 +258,84 @@ def _validate_choose_wrong_option_probability(
         )
 
 
+def validate_noise_level_proportions(
+    configuration_tree: ConfigTree, dataset: Dataset, user_filters: List[Tuple]
+) -> None:
+    """
+    Validates that the noise levels provided do not exceed the allowable proportions from the
+    metadata proportions file. If the provided noise levels are higher than the allowable proportions
+    then throw a warning to the user and adjust the noise level to the allowable proportion.
+    """
+    # TODO: update file and filepath
+    metadata_proportions = pd.read_csv(paths.METADATA_PROPORTIONS)
+    dataset_proportions = metadata_proportions.loc[
+        metadata_proportions["dataset"] == dataset.name
+    ]
+    # Set default values for state and year
+    if dataset.name == metadata.DatasetNames.SSA:
+        state = "USA"
+    else:
+        # Note: This is a shortcoming of our current approach to user warnings and will be fixed
+        # with a future release/our next data upload. We do not have a way to get state in a
+        # case where the user is not filtering on state because they either are using the sample
+        # data or state (Rhode Island) data.
```

Comment:
Is it worth making a ticket to revisit this and mention it in the comment?

Reply:
With how this is implemented a data upload will fix this issue. The comment would not be necessary then.